packages/react-components/src/components/DomainPermissions.tsx

```diff
@@ -32,6 +33,10 @@ export interface DomainPermissionsProps {
    * Action to be taken by the more help link. Possible to send to external page or show other modal.
    */
   onTroubleshootingClick: () => void;
+  /**
+   * Action to be taken by the Allow Access button.
+   */
+  onAllowAccessClick?: () => void;
```

Comment:
is there a way that this button can allow permissions? or does this trigger a permissions request again?

Reply:
As this is a component, it's up to Contoso on how they will action it.
However, the idea would be to activate the handler call for "askDevicePermission" and to query for devices.